components/SearchBar/index.js

```diff
@@ -0,0 +1,45 @@
+import styled from "styled-components";
+
+export default function SearchBar({ setSearchInput }) {
+  function inputHandler(event) {
+    const lowerCasedInput = event.target.value.toLowerCase();
+    setSearchInput(lowerCasedInput);
+  }

```

Comment:
I would move this function one level higher.

In general it is a good practice to keep the setter functions of a state close to where you initialize the state.
It keeps your code cleaner and also easier to maintain.

My suggestion has a few adaptations as its consequence:

```suggestion

```